### ISSUE TITLE
Question about custom stoploss code

### ISSUE THREAD

**Issue by thunderwin** [freqtrade/freqtrade#8796]
this custom stop-loss code seems only works on LONG ? what about SHORT ?

```
    if stoploss_price < current_rate:
            return stoploss_from_absolute(stoploss_price, current_rate, is_short=trade.is_short)
```

very confuse about this one.

```
class AwesomeStrategy(IStrategy):

    def populate_indicators(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
        # <...>
        dataframe['sar'] = ta.SAR(dataframe)

    use_custom_stoploss = True

    def custom_stoploss(self, pair: str, trade: 'Trade', current_time: datetime,
                        current_rate: float, current_profit: float, **kwargs) -> float:

        dataframe, _ = self.dp.get_analyzed_dataframe(pair, self.timeframe)
        last_candle = dataframe.iloc[-1].squeeze()

        # Use parabolic sar as absolute stoploss price
        stoploss_price = last_candle['sar']

        # Convert absolute price to percentage relative to current_rate
        if stoploss_price < current_rate:
            return stoploss_from_absolute(stoploss_price, current_rate, is_short=trade.is_short)

        # return maximum stoploss value, keeping current stoploss price unchanged
        return 1
```

**Comment by xmatthias** [comment]
Please get used to filling out the issue template.  Those who frequently omit the template may have their issues closed without comment.
We're asking some very basic questions - by deleting the issue template, you're explicitly choosing not to answer our questions.

Filling out the template helps reduce our workload, and we greatly appreciate that.


Please consider this your last warning. 
I've pointed this out in every issue you opened - the next issue without issue template will be closed without further comment.

---

Assuming you found this in a sample in the documentation (though you fail to provide any reference where this is actually from) - then it should be clear that they're examples you're expected to improve upon.

They may not be complete or working in all cases - but that's not what the samples aim for, either.
if you think it should be complete and work for shorts, too - best open a PR improving the existing documentation example.

**Comment by xmatthias** [comment]
Closing this as i think your question has been answered.

Feel free to comment below / reopen if it's still unclear.
